### PR TITLE
Remove BGP functionality and dependency

### PR DESCRIPTION
**Pull request by unix1**
This PR removes existing BGP functionality and dependencies. This "fixes" (by removing functionality) error described in #12. Note: some whitespace and linting also made it through.

### CODE REVIEW

File: DNSAuth/config.go [hunk 1]
```diff
@@ -2,15 +2,13 @@ package main
...
-	"github.com/Packet-Clearing-House/DNSAuth/DNSAuth/bgp"
 )
 
 type Config struct {
-	BGP *bgp.BGPConf
-	CustomerDB string `cfg:"customer-db; required; "`
-	CustomerRefresh int `cfg:"customer-refresh; 24; "`
-	InfluxDB string `cfg:"influx-db; required; "`
-	WatchDir string `cfg:"watch-dir; required; "`
+	CustomerDB      string `cfg:"customer-db; required; "`
```
Comment (Ths2-9Y-LqJt6) [inline DNSAuth/config.go]: No biggie, but try and keep formatting changes to at least their own commit, if not their own PR like you did in #22 ;)

**Comment by Ths2-9Y-LqJt6** [comment]
checking your fork and branch - all is well.  merged!

File: DNSAuth/config.go (hunk 1 above)
Comment (Ths2-9Y-LqJt6) [reply DNSAuth/config.go]: Oh! I see you said, "Note: some whitespace and linting also made it through.".  While good to call, still better to do in a separate commit - most important - still no biggie ;)

Comment (unix1) [reply DNSAuth/config.go]: :+1:
